### PR TITLE
Update Visually Hidden Text README.md

### PR DESCRIPTION
**Pull request by bcmn**
Follow-up to #918 

I'd hoped to run the README past Catharine when it reached the UX/a11y review section, but it was merged pretty quickly! Catharine had a few notes on things to clarify, so I've tried to incorporate them.

- [ ] ~Tests added for new features~
- [ ] ~Test engineer approval~

### CODE REVIEW

File: src/app/components/VisuallyHiddenText/README.md [hunk 1]
```diff
@@ -4,13 +4,15 @@
...
+Avoid using this component with text that would cause unnecessary repetition. For example, there is no need to add visually hidden text stating "Image" next to an `img` tag, as assistive tech already explicitly communicates the presence of an image, and this would likely only prove confusing or irritating to users.
+
+Similarly, when adding hidden text that will occur partway through an article, avoid using phrases that may imply that the main body is over. Suddenly hearing a phrase like "Read more on ___" may suggest to users that they have finished the article. To mitigate this, you may wish to wrap the content in an [`aside` ARIA landmark](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/landmarks/complementary.html) or include a ["skip link"](https://www.w3.org/TR/WCAG20-TECHS/G1.html), giving users the option to skip back to the main body before the assistive technology reads the full content of the interjection.
```
Comment (ChrisBAshton) [inline src/app/components/VisuallyHiddenText/README.md]: Markup is borked by the example text, so:

```suggestion
Similarly, when adding hidden text that will occur partway through an article, avoid using phrases that may imply that the main body is over. Suddenly hearing a phrase like "Read more on [subject]" may suggest to users that they have finished the article. To mitigate this, you may wish to wrap the content in an [`aside` ARIA landmark](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/landmarks/complementary.html) or include a ["skip link"](https://www.w3.org/TR/WCAG20-TECHS/G1.html), giving users the option to skip back to the main body before the assistive technology reads the full content of the interjection.
```

**Review by ChrisBAshton — CHANGES_REQUESTED** [review]
Otherwise looks good!

**Review by radiocontrolled — DISMISSED** [review]

**Review by radiocontrolled — APPROVED** [review]
👍

**Review by sareh — APPROVED** [review]
👍 Looks good.

**Review by ChrisBAshton — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**